docs/design/chart_format.md

```diff
@@ -234,7 +210,7 @@ $ helm release -r 1.1.2 ./nginx/
 A _Chart Repository_ is a place where _released copies_ of one or more charts may reside. (Unreleased charts are explicitly disallowed.) A Helm-DM Chart Repository is analogous to a Debian package repository or a Ruby Gems repository. It is a remote storage location accessible over a well-documented protocol (HTTP(S), and following fixed structural conventions.
```

Comment:
We might want to relax the policy of disallowing unreleased charts. Either that, or we'll have to implement `helm chart release` for MVP and use it to release the charts that are currently sitting in `gs://kubernetes-charts` and friends.